engines/dradis-api/spec/requests/dradis/ce/api/v1/evidence_spec.rb

```diff
@@ -229,7 +229,7 @@
         end
 
         context "with params for an invalid evidence" do
-          let(:params) { { evidence: { content: "a"*65536 } } } # too long
+          let(:params) { { evidence: { issue_id: '' } } }
```

Comment:
This change is here because to meet the new maximum to cause an error would require a ~4gb txt file saved in the repo, or 38 seconds to generate that many a's.